proxy/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM scratch
+MAINTAINER Weaveworks Inc <help@weave.works>
+WORKDIR /home/weave
+ADD ./proxy /home/weave/
+ENTRYPOINT ["/home/weave/proxy", "-L", ":12375"]
+CMD ["-H", "unix:///var/run/docker.sock"]
```

Comment:
I gather the proxy is actually embedded in the weaveexec image, so this file should go.